chef/cookbooks/network/attributes/default.rb

```diff
@@ -18,8 +18,11 @@
   default[:network][:base_pkgs] = ["bridge-utils",
                                    "vlan"]
   default[:network][:ovs_pkgs] = ["openvswitch",
```

Comment:
Makes it sense to remove the line break? The intention is anyway wrong so I prefer removing the line break or
```ruby
default[:network][:ovs_pkgs] = [
  "openvswitch",
  "openvswitch-switch"
]
```